src/plugins/vis_type_vega/public/vega_view/vega_map_view.js

```diff
@@ -54,8 +54,9 @@ export class VegaMapView extends VegaBaseView {
     if (!mapOptions) {
       this.onWarn(
         i18n.translate('visTypeVega.mapView.mapStyleNotFoundWarningMessage', {
-          defaultMessage: '{mapStyleParam} was not found',
-          values: { mapStyleParam: `"mapStyle":${mapStyle}` },
+          defaultMessage:
+            '{mapStyleParam} was not found. Set to "default" to use Kibana\'s default base map layer or set to "false" to disable base layer',
```

Comment:
How about listing out EMS layer ids when EMS is enabled?

Reply:
I can do it, but from the other hand I don't like how our current vega warning messages looks like. It's just a text without any formatting. I would not like to overload these messages with this information. But if you insist I'll do it.

Reply:
far enough, maybe just putting the EMS ids in the docs then